src/core/lwip/tcp_in.c

```diff
@@ -1375,7 +1376,7 @@ static void tcp_receive(struct tcp_pcb *pcb, tcp_in_data *in_data)
                adjust the ->data pointer in the seg and the segment
                length.*/
 
-            off = pcb->rcv_nxt - in_data->seqno;
+            u32_t off = pcb->rcv_nxt - in_data->seqno;
```

Comment:
Are we sure that pcb->rcv_nxt = in_data->seqno. What if malicious app sends invalid packet with bigger seqno?

Reply:
I believe there is a check for this somewhere at the top of the function.

Reply:
How do we handle here wrap arounds? even before the change?